src/dl.cpp

```diff
@@ -106,15 +106,15 @@ dl_error_t DL_DLL_EXPORT dl_instance_load_inplace( dl_ctx_t       dl_ctx,
 
 struct CDLBinStoreContext
 {
-	CDLBinStoreContext( uint8_t* out_data, size_t out_data_size, bool is_dummy )
+	CDLBinStoreContext( uint8_t* out_data, size_t out_data_size, bool is_dummy, dl_allocator alloc )
+	    : written_ptrs(alloc)
 	{
 		dl_binary_writer_init( &writer, out_data, out_data_size, is_dummy, DL_ENDIAN_HOST, DL_ENDIAN_HOST, DL_PTR_SIZE_HOST );
-		num_written_ptrs = 0;
 	}
 
 	uintptr_t FindWrittenPtr( void* ptr )
 	{
-		for( int i = 0; i < num_written_ptrs; ++i )
+		for (size_t i = 0; i < written_ptrs.Len(); ++i)
```

Comment:
Sorry about the style changes, Visual Studio does these automatically for me using some default clang-format style since this library is missing a custom one.

Reply:
yes... I should setup a clang-fmt file.